src/main/java/lotto/LottoTicketGenerator.java

```diff
@@ -0,0 +1,31 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+import lotto.domain.LottoTicket;
+import lotto.domain.LottoTicketBall;
+
+public class LottoTicketGenerator {
+
+    private static final int PRICE_PER_TICKET = 1000;
+    private LottoTicketBall lottoTicketBall = new LottoTicketBall();
```

Comment:
상수와 멤버변수 간 개행을 주면 좋겠네요 
```suggestion
    private static final int PRICE_PER_TICKET = 1000;
    
    private LottoTicketBall lottoTicketBall = new LottoTicketBall();
```